python/lsst/summit/utils/consdbClient.py

```diff
@@ -77,6 +77,24 @@ def _check_status(r: requests.Response) -> None:
         raise e
 
 
+def clean_url(resp: requests.Response, *args, **kwargs) -> requests.Response:
+    """Parse url from response and remove netloc portion.
+
+    Set new url in response and return response
+
+    Parameters
+    ----------
+    resp : `requests.Response`
+        The response that could contain a URL with tokens
+    """
+    url = urlparse(resp.url)
+    short_user = url.username[:2] if url.username is not None else "**"
```

Comment:
I think it's a bit more useful to have
```
short_user = f"{url.username[:2]}***" if url.username is not None else ""
short_pass = f":{url.password[:2]}***" if url.password is not None else ""
netloc = f"{short_user}{short_pass}@{url.hostname}"
```
That way, empty usernames and passwords are properly reflected and the password component is only included if it is actually present.